docs/docker/docker-images.md

```diff
@@ -0,0 +1,218 @@
+# Docker Images
+
+## Building your own image
+Please refer [Build your own image](https://docs.docker.com/mac/step_four/) for more reference.
+
+The below steps only provide instructions for creating and building docker images.
+
+### Build image using docker file
+The `docker build -t ng2-book-finder .` command takes the `Dockerfile` in the current directory, and builds an image called `ng2-book-finder` on your local machine.
```

Comment:
add a link to ng2-book-finder  repo  https://github.com/introduction-angular2/ng2-book-finder